src/screens/EditWordScreen/EditWordScreen.js

```diff
@@ -57,37 +57,33 @@ class EditWordScreen extends Component {
     }
   }
 
-  generateTranslation = translationList => translationList.map((word) => {
-    const deleteCurrentValue = () => this.deleteWordOrExample(word, EDIT_TYPES.word)
+  generateItem = (itemsList, type) => {
+    const currentStyle = {}
```

Comment:
Why do you need an additional variable?
```
First proposal:
style={isTranslation ? styles.translationItem : styles.exampleItem}
```
```
The styles are pretty much the same. You could create one style for `EDIT_TYPES.word` and another style with differences.
Second proposal:
style={[
              styles.translationItem,
              isExample && styles.exampleItem
]}
```

Reply:
Because this variable will be created only once and the check will be made once. For your advice the check will be made for each element of the array